modules/ztp-setting-managed-bare-metal-host-kernel-arguments-gitops-workflow.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-arguments_{context}"]
+= Setting managed bare-metal host kernel arguments
```

Comment:
I think you should differentiate these section titles to ensure that anyone who lands here from Google doesn't get confused. 

Maybe "Adding host kernel arguments to the Discovery ISO using GitOps ZTP" and "Adding host kernel arguments to the Discovery ISO for manual installations"